cypress/e2e/supervision/clients/record-client-death-confirmation.cy.js

```diff
@@ -0,0 +1,96 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createAClient();
+});
+
+describe(
+  "Successfully record client death notification",
+  { tags: ["@supervision-core", "@client", "@client-record-death-confirmation", "@smoke-journey"] },
+  () => {
+    const dateOfDeath = "16/03/2023";
+    const dateDeathCetificateReceived = "16/03/2023";
+    const dateNotified = "16/03/2023";
+    const notifiedBy = "Deputy";
+    const howNotified = "Email";
+    const dateOfDeathIncorrect = "Test date";
+    it(
+      "Records a client's death confirmation successfully when populating all fields",
+      () => {
+        cy.get("@clientId").then((clientId) => {
+          cy.visit(`/supervision/#/clients/${clientId}/record-death`);
+        });
+
+        cy.get("#record-death").as("record-death-panel");
+        cy.get("@record-death-panel").within(() => {
+          cy.contains("Proof of death received")
+            .closest(".fieldset")
+            .contains("Yes")
+            .click();
+          cy.get('input[name="dateOfDeath"]').clear().type(dateOfDeath);
+          cy.get('input[name="dateDeathCertificateReceived"]').clear().type(dateDeathCetificateReceived);
+          cy.get('input[name="dateNotified"]').clear().type(dateNotified);
+          cy.get('[name="notifiedBy"]').select(notifiedBy);
+          cy.contains("How was the OPG notified?")
+            .closest(".fieldset")
+            .contains(howNotified)
+            .click();
+          cy.window()
+            .its("tinyMCE")
+            .its("activeEditor")
+            .its("initialized", {timeout: 2000});
+          cy.window().then((win) => {
```

Comment:
You might not need to do this as paste data, some tests do that to check paste functionality, but here you might be able to just set editor.dom content.

Reply:
Sorry Matt that's actually not my test, I think it's just in here as I did a rebase. I've only amended the add-deputy one.

Reply:
Huh, weird. Did you run prettier or some kinting over the codebase? Might have reformatted the file and created the diff from main.